requirements-selftests.txt

```diff
@@ -3,8 +3,6 @@
 setuptools>=18.0.0
 # sphinx (doc build test)
 Sphinx>=1.3b1
-# flexmock (some unittests use it)
-flexmock>=0.9.7
```

Comment:
this should be the last commit (you need flexmock until you fix all the places)

Reply:
And it is! GH is messing with the order because of the date.  I tried a couple of times to do an `--ignore-date`, but something is acting funny.